examples/window_zoom.rs

```diff
@@ -0,0 +1,23 @@
+use dioxus::desktop::use_window;
+use dioxus::prelude::*;
+
+fn main() {
+    dioxus::desktop::launch(app);
+}
+
+fn app(cx: Scope) -> Element {
+    let window = use_window(&cx);
+
+    let level = use_state(&cx, || 1.0);
+
+    window.set_zoom_level(*level.get());
```

Comment:
looks like this sets the zoom level on every re-render?

in this example its kinda fine, but people might copy-paste it into their own work and that won't be great

may i suggest calling this inside the event handling closure?